samples/tokenCredentialSamples/AuthenticationCodeFlow/package.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "authentication_code_flow_sample",
+	"version": "1.0.0",
+	"description": "sample of an auth provider using token credentials",
+	"scripts": {
+		"start": "npm install && npx ts-node src/index.ts"
+	},
+	"dependencies": {
+		"@azure/identity": "^1.1.0",
+		"express": "^4.17.1",
+		"open": "^7.2.1",
+		"qs": "^6.9.4",
+		"ts-node": "^9.0.0",
+		"typescript": "^4.0.3"
```

Comment:
shouldn't our sdk be in the list of deps, but with a relative path instead?

Reply:
Done